api-server/apiserver.py

```diff
@@ -1,9 +1,15 @@
 import os
-from flask import Flask, json, abort, jsonify, redirect, url_for
-import flask
-from semver import VersionInfo
 import subprocess
 
+from flask import Flask, json, abort, jsonify
+from semver import VersionInfo
+
+
+if os.environ.get('SENTRY_DSN'):
```

Comment:
This should not be necessary, you can also just always monkeypatch. It just won't do anything

Reply:
Ah, good, thanks